### ISSUE TITLE
cmpo_loss error?

### ISSUE THREAD

**Issue by qianfangjj** [YuriCat/MuesliJupyterExample#3]
Hello, Is there some error with the cmpo_loss code? As there has a minus sign, but the formula in the article do not has minus sign.
cmpo_loss += -torch.mean(sea[:, t] / sz[:, t] * torch.log(ps[t].gather(1, sa[:, t])), dim=1).sum()

**Comment by YuriCat** [comment]
Massive thanks for finding that point!
I believe the bug is in the article, and this is what I got after transforming from equation 9 in it.
I hope you check it too!

**Comment by qianfangjj** [comment]
You are right, Equation 11 should have the minus sign. I have asked the author. 
Thank you for your code. It helps me a lot.
